### ISSUE TITLE
Mods not showing

### ISSUE THREAD

**Issue by bunniebopp** [mkinfrared/baldurs-gate3-mod-manager#36]
I have my mods installed through this mod manager and it shows they're in the folder but when I open the game, they're not there?
<img width="320" alt="Screenshot 2023-10-08 at 10 24 31 am" src="https://github.com/mkinfrared/baldurs-gate3-mod-manager/assets/147274384/469c3ac9-535f-457a-b2cd-26aac9dd0f0e">
<img width="392" alt="Screenshot 2023-10-08 at 10 24 50 am" src="https://github.com/mkinfrared/baldurs-gate3-mod-manager/assets/147274384/e1032be2-79af-49ec-a811-b071ac26f893">
Was wondering if you could help, thank you!

**Comment by DIXAK** [comment]
Hi, if you find a solution please let me know! Really need the nomorespiders mod

**Comment by mkinfrared** [comment]
Hi, which one exactly doesn't work for you?

**Comment by Timodej** [comment]
None work for me either. Do we still need to manually edit the modsettings.lsx file?

**Comment by LiviuSL** [comment]
Yes, wanted to say thanks as well! Great to see that this is being done for Mac as well. However, I'm having a similar issue with the one above. 

My installedmods.json file is showing the entries for the three mods I installed with the mod manager - ModFixer | ImprovedUI | RingOfElderGods (not sure if relevant but these don't have a info.json file) - , however, the ones I previously had (which are similar to the ones posted in the above screenshot) aren't showing there. When I load the game, I can see the mods that were in modsettings.lsx ingame, but the ones I installed with the mod manager aren't showing at all.

**Comment by mkinfrared** [comment]
Probably it has a different `info.json` structure. You can check this issue #33. I released a new update to support that mod installation.

**Comment by mkinfrared** [comment]
@LiviuSL which mods are not working for you? Can you add links to those mods?

**Comment by LiviuSL** [comment]
Yes, I tried adding this one: https://www.nexusmods.com/baldursgate3/mods/1321?tab=files.

**Comment by mkinfrared** [comment]
Yes, I see that this mod comes with pak file only. Did appear in your Mods folder? If yes, then it probably has to do something with the mod. Some people posted that mods might now work due to a new patch. Try to check the posts for that mod.

**Comment by LiviuSL** [comment]
Thank you for looking into it! Yeah - the mod is in the mods folder. The manager is definitely doing that well. I'll follow this project, looking forward to using it! :) Good job!
